CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ All notable changes to this project are documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased](https://github.com/airbnb/epoxy-ios/compare/0.3.0...HEAD)
+## [Unreleased](https://github.com/airbnb/epoxy-ios/compare/0.4.0...HEAD)
```

Comment:
oops missed this when I released 0.4.0!

Reply:
all good! we can eventually write a script that does this I think